src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: roboto, sans-serif;
+}
```

Comment:
use classes
```suggestion
body {
  margin: 0;
  font-family: Roboto, sans-serif;
}
```

Reply:
I try to use tag body in styling only for reseting margins and for font family.